### PR TITLE
Update of oc.jsx

### PR DESCRIPTION
**Pull request by Quent-in**
Now similar to the English one. I'll update oc.yml within a week.

### CODE REVIEW

**Comment by Quent-in** [comment]
I don't really get why the Travis Cl build failed =/

**Comment by mcmarto** [comment]
The CI build failed because a number of lines in oc.jsx are missing the trailing comma.

**Comment by Quent-in** [comment]
I'm sorry but I don't know what it is =/

File: app/assets/javascripts/components/locales/oc.jsx [hunk 1]
```diff
@@ -1,128 +1,177 @@
...
-  "column.home": "Acuèlh",
-  "column.community": "Fil public local",
-  "column.public": "Fil public global",
-  "column.notifications": "Notificacions",
+  "account.unblock": "Desblocar @{name}",
+  "account.unfollow": "Quitar de sègre",
+  "account.unmute": "Quitar de rescondre @{name}",
+  "boost_modal.combo": "Podètz picar {combo} per passar aquò lo cop que ven",
   "column.blocks": "Personas blocadas",
+  "column.community": "Fil public local"
```
Comment (ashfurrow) [inline app/assets/javascripts/components/locales/oc.jsx]: This line needs a comma `,` at the end.

**Review by ashfurrow — CHANGES_REQUESTED** [review]
I think that's the only change needed to get Travis to work.

File: app/assets/javascripts/components/locales/oc.jsx (hunk 1 above)
Comment (mcmarto) [inline app/assets/javascripts/components/locales/oc.jsx]: Lines 20, 29 (shown above) and 167 are all missing the comma at the end of the line.

**Comment by Quent-in** [comment]
It should be okay now :)

File: app/assets/javascripts/components/locales/oc.jsx (hunk 1 above)
Comment (Quent-in) [inline app/assets/javascripts/components/locales/oc.jsx]: I changed it, I hope it's okay now.

**Pull request closed without merging**